pyproject.toml

```diff
@@ -17,7 +17,9 @@ dependencies = [
     "colorama",
     "requests",
     "twine >= 1.6.0",
+    "build >= 0.6.0.post1",  # 0.6.0 wasn't compatible with Python 3.7
```

Comment:
Actually... that can be a separate decision.

Reply:
I am fine with dropping Python 3.7 support: this is EOL since a few weeks.
We should only do that in a major release.  But the changes from this PR seem pretty major already, so I think that is the plan. :-)